opendatahub-io/data-science-pipelines-operator
# Added argument to configure log levels for the operator and defined `…

## The issue resolved by this Pull Request:
Resolves https://github.com/opendatahub-io/data-science-pipelines-operator/issues/319

## Description of your changes:
* Added argument to configure log levels for the operator
* Set `info` as default.
* Added documentation

### Regarding the documentation
I added it at the end of README.md. I'm not sure it's the best/correct place. Please, let me know if I can move it to somewhere else.




## Testing instructions

Set the desired log level as in the docs, deploy it, and check the operator's logs.

## Checklist
